test/transforms.jl

```diff
@@ -685,6 +685,22 @@
     @test t == tₒ
   end
 
+  @testset "Coerce" begin
+    x1 = [1.0, 2.0, 3.0, 4.0, 5.0]
+    x2 = [1.0, 2.0, 3.0, 4.0, 5.0]
+    x3 = [5.0, 5.0, 5.0, 5.0, 5.0]
+    t = TypedTables.Table(;x1, x2, x3)
+
+    # apply test
+    n, c = apply(Coerce(:x1=>Count, :x2=>Count), t)
+    @test elscitype(n.x1) == Count && elscitype(n.x2) == Count
+
+    # revert test
+    n, c = apply(Coerce(:x1=>Count, :x2=>Count), t)
+    tₒ = revert(Coerce(:x1=>Count, :x2=>Count), n, c)
+    @test t == tₒ
```

Comment:
Are you sure this test is checking what we want? Try to write a more explicit test like you did for the apply function. We need to test that the columns have the original type. The test with == is only checking the values in the table.